framework/src/postprocessors/ElementW1pError.C

```diff
@@ -34,7 +34,8 @@ ElementW1pError::ElementW1pError(const InputParameters & parameters) :
 Real
 ElementW1pError::getValue()
 {
-  return std::pow(ElementIntegralPostprocessor::getValue(), 1./_p);
```

Comment:
BASE CLASS!

@dschwen's correct, while your new statements might be equivalent now, if we change the BASE CLASS, things won't flow downstream anymore.